src/app/core/services/user/user.service.ts

```diff
@@ -72,8 +73,8 @@ export class UserService {
       return throwError('createUser() called without required body data');
     }
 
-    if (!body.customer.type) {
-      return throwError('createUser() called without required customer type (PrivateCustomer/SMBCustomer)');
+    if (body.customer.isBusinessCustomer === undefined) {
+      return throwError('createUser() called without required customer type (isBusinessCustomerRegistration)');
     }
```

Comment:
is the feature toggle isBusinessCustomerRegistration dependant on the flag? :thinking: